.clang-tidy

```diff
@@ -66,4 +111,4 @@ CheckOptions:
   - key:   readability-identifier-naming.TemplateParameterCase
     value: CamelCase
   - key:   performance-unnecessary-value-param.AllowedTypes
-    value: [Ff]unction$
+    value: '[Ff]unction$'
```

Comment:
I assume that was the reason why it was malformed?

Reply:
Yes exactly (see my comment below).